### PR TITLE
chore(vite installation guide): add installation guide using pnpm

### PR DESCRIPTION
**Pull request by dammy001**
## Description of Problem
I figured out that there's no vite installation guide using PNPM

## Proposed Solution
This PR consists of vite installation guide using PMPM

### CODE REVIEW

File: src/guide/installation.md [hunk 1]
```diff
@@ -117,6 +117,15 @@ $ yarn
 $ yarn dev
 ```
 
+Or with PNPM:
+
+```bash
+$ pnpm create vite <project-name> --template vue
```
Comment (skirtles-code) [inline src/guide/installation.md:123]:
```suggestion
$ pnpm create vite <project-name> -- --template vue
```

This didn't work for me, testing with pnpm 6.23.1. I had to include an extra `--`, a bit like with npm 7+. The Vite docs seem to show the same thing: https://vitejs.dev/guide/#scaffolding-your-first-vite-project

**Review by skirtles-code — CHANGES_REQUESTED** [review]
Thanks for the PR!

I agree this would make a good addition. If you could review and accept my suggested changes I think we can get this merged.

File: src/guide/installation.md [hunk 2]
```diff
@@ -117,6 +117,15 @@ $ yarn
 $ yarn dev
 ```
 
+Or with PNPM:
```
[the remark this answers is not in the document]
Comment (dammy001) [reply src/guide/installation.md:120]: Okay. Thanks

**Comment by dammy001** [comment]
Done ✅

**Review by NataliaTepluhina — APPROVED** [review]

**Pull request merged**